src/CommonDBVisible.php

```diff
@@ -309,17 +309,17 @@ public function showVisibility()
                         Html::showMassiveActionCheckBox('Group_' . $this::getType(), $data["id"]);
                         echo "</td>";
                     }
-                    echo "<td>" . Group::getTypeName(1) . "</td>";
+                    echo "<td>" . htmlspecialchars(Group::getTypeName(1)) . "</td>";
 
                     $names   = Dropdown::getDropdownName('glpi_groups', $data['groups_id'], 1);
-                    $entname = sprintf(
+                    $entname = htmlspecialchars(sprintf(
                         __('%1$s %2$s'),
                         $names["name"],
                         Html::showToolTip($names["comment"], ['display' => false])
-                    );
+                    ));
                     if ($data['entities_id'] !== null) {
                         $entname = sprintf(
-                            __('%1$s / %2$s'),
+                            __s('%1$s / %2$s'),
                             $entname,
                             Dropdown::getDropdownName(
```

Comment:
`Dropdown::getDropdownName()` returns a non encoded string, it should be escaped too.
```suggestion
                            htmlspecialchars(Dropdown::getDropdownName(
```

I cannot add a sugesstion for the closing parenthesis :/.

Reply:
Fixed